packages/sanity/src/core/form/inputs/arrays/ArrayOfObjectsInput/List/ListArrayInput.tsx

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable react/jsx-handler-names */
 import {Card, Stack, Text} from '@sanity/ui'
 import React, {useCallback, useMemo} from 'react'
+import {isEqual} from 'lodash'
```

Comment:
💭 should we use `react-fast-compare` instead since it's supposedly faster?

Reply:
Actually I think `shallow-equals` makes more sense here and is a little bit clearer. I doubt there will be any noticeable differences between react-fast-compare and shallow-equals here since they both essentially do the same thing when comparing arrays of strings